### ISSUE TITLE
Add khmer language support to the app

### ISSUE THREAD

**Issue by ifoche** [EyeSeeTea/malariapp#822]

**Comment by ifoche** [comment]
We are going to be showing khmer text (for Cambodia team) soon, so we need to prepare the app for that. In pictureapp they've already done it, and commits that shows how to include that library are [this](https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2FEyeSeeTea%2Fpictureapp%2Fcommit%2F5a1b040c1e1bbfab69f2f8c43a17ed75d4e6271a&sa=D&sntz=1&usg=AFQjCNHa35zMtygoxMS8ZZ6vZRF4nFlQEA) and [this](https://www.google.com/url?q=https%3A%2F%2Fgithub.com%2FEyeSeeTea%2Fpictureapp%2Fcommit%2Fd79732e821096534ce30f6c90d7c34a5507ff343&sa=D&sntz=1&usg=AFQjCNEYvzarFoyJF3oouTkW0_RzQ2gJOQ)

**Comment by adrianq** [comment]
@ifoche @idelcano Some thoughts: this card is only about devices below 5.x (i.e. 4.x), as in 5.x and 6.x Khmer is supported by default.  Cambodian people usually have their devices prepared for Khmer alphabet so they won't need any change in the code. This change will only be needed for non-Cambodian mobile phones when dealing with Cambodian alphabet (like our emulators and phones). Is this something we would like to take care? 

We had to do it for some data elements in pictureapp (questions). Questions could only be found in the database and as you know  we (in the csv) and Oslo (in the dhis api) do not support multilanguague. Therefore, as we want to display something in the app (and not show a blank field)  we added that special font.
